scripts/install.sh

```
@@ -632,6 +640,169 @@ download_kernel() {
+    #
+    # 1. /var/lib/zwrm pinned to a different (or unresolvable) device →
+    #    hard conflict; the next boot would mount the wrong thing.
+    if zwrm_fstab_conflicts_with "$dev"; then
+        local conflict_src
+        conflict_src="$(fstab_zwrm_source)"
+        fatal "/etc/fstab already mounts /var/lib/zwrm from another source (${conflict_src:-unresolved}); remove or fix that line, then re-run"
+    fi
+    # 2. The device must not be persisted to ANY other mountpoint — one
+    #    block device cannot mount at two targets. This also covers the
+    #    case where it is listed at /var/lib/zwrm AND elsewhere.
+    local dev_targets other_targets=""
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)